dev/ldap/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3.3'
+services:
+    openldap:
```

Comment:
This file could go directly in the docker directory at `docker/docker-compose.yml`, on the expectation that we'll put more containers in here over time. (e.g. postgresql to test the database secrets engine)